src/sentry/static/sentry/app/components/search/index.jsx

```diff
@@ -94,6 +95,13 @@ class Search extends React.Component {
 
     if (to.startsWith('http')) {
       const open = window.open();
+      if (open === null) {
+        IndicatorStore.addError(
+          t('Unable to open search result (a popup blocker may have caused this).')
```

Comment:
I have some advice on how to do this without being blocked and avoiding `window.open`; we could talk offline.

Reply:
Sure. I looked at a few solutions but couldn't find one that would open in a new tab.

Reply:
I'm not going to get too involved on this one @benvinegar. We can circle back at some point and turn these into anchors and look closer into the key handling stuff.